deploy/user-default-role.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: sandbox-users
+  name: sandbox-user
```

Comment:
nit: We'll want to update the documentation to reflect this change as well

Reply:
Documentation should be updated